### ISSUE TITLE
/resize aspect ratio is not being maintained

### ISSUE THREAD

**Issue by foobargeez** [h2non/imaginary#232]
I have an image with 1529x2500 dimensions and I am calling imaginary as follows:

```https://<imaginary host>/resize?file=test/test.jpg&width=960&stripmeta=true&quality=70```

The output image has 960x2500 dimensions and I am expecting the output image dimensions to be 960x1570.  Any idea how to make imaginary honor the aspect ratio?

Thanks!

**Comment by foobargeez** [comment]
BTW, if imaginary doesn't (want to) honor the aspect ratio, I can pass the height too (which is what I do) but the doc says, imaginary takes aspect ratio into account, so just making sure I understand what to expect.

**Comment by Dynom** [comment]
Imaginary should actually respect the aspect-ratio, but this part is a tad bit confusing. Try specifying the `nocrop=true` parameter and omit the height.

This works for me:

**Comment by foobargeez** [comment]
Specifying `nocrop=true` is respecting the aspect ratio.  This workaround works for me (unless this is default behavior in which case, we should clarify in the doc).

**Comment by Dynom** [comment]
I agree it is and I think it can even be considered a bug, but we need to dive into that first. With image manipulation things aren't always that simple :-)
